ios/browser/brave_web_client.mm

```diff
@@ -44,10 +44,14 @@
 
   schemes->standard_schemes.push_back(kBraveUIScheme);
   schemes->secure_schemes.push_back(kBraveUIScheme);
+
+  schemes->standard_schemes.push_back(kChromeUIUntrustedScheme);
+  schemes->secure_schemes.push_back(kChromeUIUntrustedScheme);
 }
 
 bool BraveWebClient::IsAppSpecificURL(const GURL& url) const {
-  return ChromeWebClient::IsAppSpecificURL(url) || url.SchemeIs(kBraveUIScheme);
+  return ChromeWebClient::IsAppSpecificURL(url) ||
+         url.SchemeIs(kBraveUIScheme) || url.SchemeIs(kChromeUIUntrustedScheme);
```

Comment:
doesn't this enable mojo bindings? https://source.chromium.org/chromium/chromium/src/+/main:ios/web/webui/web_ui_messaging_java_script_feature.mm;l=47